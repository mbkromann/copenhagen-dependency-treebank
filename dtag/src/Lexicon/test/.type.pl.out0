a1: super=[b c __undef__] lvars=[]
a2: super=[b c __undef__] lvars=[]
a3: super=[b c __undef__] lvars=[]
