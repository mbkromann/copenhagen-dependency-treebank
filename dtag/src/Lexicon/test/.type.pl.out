ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
a1: super=[b c __undef__] lvars=[]
a2: super=[b c __undef__] lvars=[]
a3: super=[b c __undef__] lvars=[]
